app/core/vueRouter.js

```diff
@@ -26,6 +26,17 @@ export default function getVueRouter () {
           component: () => import(/* webpackChunkName: "ParentsView" */ 'app/views/landing-pages/parents/PageParents'),
           props: (route) => ({ showPremium: true, type: route.query.type })
         },
+        {
+          path: '/parents-v2',
+          component: () => import(/* webpackChunkName: "ParentsView" */ 'app/views/landing-pages/parents-v2/PageParents'),
```

Comment:
with both parents being different content, might be a good idea to have a different chunkName too